Player/Player.gd

```diff
@@ -170,15 +172,22 @@ func move_jump(delta):
     play_in_air_animations()
         
 func move_grapple(delta):
+    #TODO better name and move up to other consts
+    var DOWNWARD_VELOCITY_LIMIT = 250
     var input_vector = Vector2.ZERO
     input_vector.x = Input.get_action_strength("move_right") - Input.get_action_strength("move_left")
     input_vector = input_vector.normalized()
     
-    hook_velocity = to_local(grapplingHook.tip).normalized() * CHAIN_PULL * delta
-    hook_velocity.x *= 5.65
-    if sign(input_vector.x) != sign(hook_velocity.x):
-        hook_velocity.x *= 0.7
-    velocity += hook_velocity
+    hook_velocity = Vector2.ZERO
+    if velocity.y >= DOWNWARD_VELOCITY_LIMIT:
+        print("using enhanced grapple")
+        hook_velocity = to_local(grapplingHook.tip).normalized() * CHAIN_PULL * delta * 35
```

Comment:
Is this pulling the player faster if they're at a high downward velocity to quickly compensate for changing direction?

Reply:
Yeah, exactly! I found that when falling at high enough speeds the grapple's pull was not effective enough to actually pull you up at all, and this doesn't feel fantastic. Boosting the grapple velocity allows the player to actually come out of their fall and potentially grapple to a land spot. Another solution is to cap our falling speed to allow players to react better.